package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "bedrock-web-payment",
+  "version": "0.0.1",
+  "description": "Client side utilities for bedrock-payment.",
+  "main": "lib/index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/digitalbazaar/bedrock-web-payment.git"
+  },
+  "keywords": [
+    "payment",
+    "bedrock",
+    "web",
+    "client",
+    "javascript"
+  ],
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-web-payment/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-web-payment#readme",
+  "peerDependencies": {
+    "bedrock-payment": "^0.0.1",
+    "axios": "^0.19.0"
```

Comment:
axios looks like a regular dependency to me.

Reply:
Removed Axios entirely here: https://github.com/digitalbazaar/bedrock-web-payment/pull/1/commits/0618c257af143d5a0adf0ea68f7f4f84c59f9358

Reply:
I don't think that's what we want.  IMO, consumers of services modules should not have to concern themselves with specifying an HTTP module to use.  See my other comment on this topic as well.

Reply:
Bundle size is what I'm concerned about. Axios is not a huge library, but if we didn't care about features or backwards compatibility we could use `fetch` directly or newer modules like `mappersmith`. On the backend including a node `httpsAgent` provides little overhead for the customer as they are already using node.

Reply:
Added axios as a dependency: https://github.com/digitalbazaar/bedrock-web-payment/pull/1/commits/86b77edc8ddf6279e6be7ca30b44a6dae20d1bfc